src/infection-model/rt.tsx

```diff
@@ -79,8 +79,10 @@ const getRtInputsForFacility = async (
   const dates: string[] = [];
 
   modelVersions.forEach((model) => {
+    const seconds = model.observedAt.getTime() / 1000;
+
     dates.push(
-      formatISO(fromUnixTime(model.observedAt.seconds), {
+      formatISO(fromUnixTime(seconds), {
```

Comment:
@macfarlandian - The persistence layer is now consistently returning native JavaScript Dates instead of Firestore Timestamps.  This update should function equivalently to what was previously here but to the extent it's useful to work with actual dates, feel free to modify the rt and other modeling code.

Is there a way I can test the RT code locally?  I didn't see this module exported anywhere

Reply:
in latest master it should be testable now ... you just have to enable the feature flag via environment variable (instructions in README) and restart your dev build